mix.exs

```diff
@@ -40,9 +40,9 @@ defmodule GRPC.Mixfile do
       {:gun, "~> 2.0.0-rc.2"},
       {:cowlib, "~> 2.11"},
       {:protobuf, "~> 0.10", only: [:dev, :test]},
-      {:ex_doc, "~> 0.28", only: :dev},
-      {:inch_ex, "~> 2.0", only: [:dev, :test]},
-      {:dialyxir, "~> 1.1", only: [:dev, :test], runtime: false}
+      {:ex_doc, "~> 0.28.0", only: :dev},
+      {:inch_ex, "~> 2.0.0", only: [:dev, :test]},
+      {:dialyxir, "~> 1.1.0", only: [:dev, :test], runtime: false}
```

Comment:
Why we change the version constraint?

Reply:
These are only for the lib itself (only prod deps are propagated to the client libs). 

Therefore, I made this more strict because I always prefer to be as strict as possible with dependencies

Reply:
👍